bgpd/bgp_evpn_mh.c

```diff
@@ -1175,6 +1175,12 @@ static int bgp_zebra_send_remote_es_vtep(struct bgp *bgp,
 	struct stream *s;
 	uint32_t flags = 0;
 
+	/*
+	 * Zebra does not process this message if advertise-all-vni is disabled
+	 */
+	if (!is_evpn_enabled())
```

Comment:
We don't seem to have this check for other EVPN routes. I don't have a strong opinion for or against it. Only it needs to be consistent.

Reply:
For "no advertise-all-vni" case BGP does not invoke bgp_zebra_send_remote_macip function. We just silently delete these remote type-2 routes. That is why this check is not needed in bgp_zebra_send_remote_macip function. Do you think this check can cause issues? In that case I will delete this check.

Reply:
If it is not needed (for functionality) maybe remove it from this PR. And you can bring it in later (in a separate PR) if you think it is need as an optimization.

Reply:
Looks like commit https://github.com/FRRouting/frr/pull/8154/commits/333ad64b874a6527d3109c5a112a2b90225bd359 also fixes the same issue?

Reply:
@adharkar Yes, I missed that altogether, sorry. I had made the changes last year but only made it upstream now (all MH changes are now upstream). Can you please check if this PR is still needed?